src/api/config/routes/webui_routes.rb

```diff
@@ -51,9 +51,14 @@
 
     resources :package, only: [:index], controller: 'webui/package', constraints: cons
 
+    controller 'webui/package' do
+      get 'package/edit/:project/:package' => :edit, constraints: cons, as: 'edit_package'
+      patch 'package/update' => :update, constraints: cons
```

Comment:
would be possible to make it more resourceful and bring it inside of project?

Reply:
I've been looking into this and before that we need to refactor a lot of existing routes. For now I'll let them as they are